homeassistant/components/seventeentrack/config_flow.py

```diff
@@ -0,0 +1,153 @@
+"""Config flow for SeventeenTrack."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from seventeentrack import Client as SeventeenTrackClient
+from seventeentrack.errors import SeventeenTrackError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME, CONF_SCAN_INTERVAL, CONF_TOKEN
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import aiohttp_client
+
+from .const import (
+    CONF_SHOW_ARCHIVED,
+    DEFAULT_NAME,
+    DEFAULT_SCAN_INTERVAL,
+    DEFAULT_SHOW_ARCHIVED,
+    DOMAIN,
+)
+from .errors import AuthenticationError
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default=DEFAULT_NAME): str,
```

Comment:
Would it work to use the account ID as default title and device name and not ask for a name here? The user can rename the config entry title and device name anyway.